tensorflow/core/common_runtime/eager/execute.cc

```diff
@@ -596,7 +596,7 @@ Status GetOrCreateKernelAndDevice(
 
       DVLOG(1) << "Placer place op [" << op->Name()
                << "] on device: " << device->name();
-      DVLOG(4) << "Available kernels for " << op->Name() << "are "
+      DVLOG(4) << "Available kernels for " << op->Name() << " are"
```

Comment:
Shouldn't this be `" are "`? Could you double-check?

Reply:
For this case, no. Because the string generated by "KernelsRegisteredForOp" has a preceeding space.
With `` are `` you will have a string like this:  `` "Available kernels for Cast are  device='CPU'" ``
Note there are 2 spaces between "are" and "device".
Either one is fine with me though...